lib/internal/bootstrap_node.js

```diff
@@ -42,6 +43,15 @@
     NativeModule.require('internal/process/warning').setup();
     NativeModule.require('internal/process/next_tick').setup();
     NativeModule.require('internal/process/stdio').setup();
+    if (browserGlobals) {
+      // Instantiate eagerly in case the first call is under stack overflow
+      // conditions where instantiation doesn't work.
+      global.console.error;
+      global.console.log;
+      global.console.time;
+      global.console.timeEnd;
+      global.console.trace;
```

Comment:
seems like `console.dir()` should be included here as well as it's not an alias?